src/redux/configureStore.js

```diff
@@ -0,0 +1,15 @@
+import { configureStore } from '@reduxjs/toolkit';
+import { combineReducers } from 'redux';
+import bookReducer from './books/books';
+import categoryReducer from './categories/categories';
+
+const rootReducer = combineReducers({
+  books: bookReducer,
+  category: categoryReducer,
+});
+
+const store = configureStore({
+  reducer: rootReducer,
+});
```

Comment:
- [ ] [Optional] When you import `configureStore` from `redux toolkit` you don't need `combineReducer` you can add all reducers like that:
```javascript
const store = configureStore({
  reducer: {
     books: bookReducer,
     category: categoryReducer,
}});
```